doc/CONTRIBUTING.md

```diff
@@ -49,12 +49,10 @@ reset VS Code then run `yarn vscode:patch`.
 
 ## Build
 
+The script we run in CI that does all the above
```

Comment:
Does "all the above" refer to the development steps? Or should this say "below" in reference to the list of what it includes below?

Reply:
no sry this was just a brainfart lol.